docs/guide/pages.md

```diff
@@ -0,0 +1,239 @@
+# Pages and page types
+
+Every page on the an Apostrophe website is assigned a **"page type"**. The page type tells Apostrophe **what template to use** to render the page and **what configuration to apply**. Configurations will often at least include the content schema for the page type.
+
+## Creating a page type
+
+Apostrophe core only includes a "Home page" type with minimal functionality. You will likely need your own page types, which you create by adding modules that extend `@apostrophecms/page-type` and instantiating them in `app.js`.
+
+```js
+// modules/default-page/index.js
```

Comment:
Is default page like, really a thing, or is it just a habit we have? Should we use a better example?